command/expand_test.go

```diff
@@ -328,6 +329,86 @@ func TestExpandSource_Do_Not_Follow_Symlinks(t *testing.T) {
 	assert.Equal(t, []string{workdirJoin}, expected)
 }
 
+func TestExpandSourceWithRawFlagTrue(t *testing.T) {
```

Comment:
```suggestion
func TestExpandSourceWithRawFlag(t *testing.T) {
```

Reply:
changed to `TestExpandSourceWithRawFlag`.